modules/osint/dns_search.py

```diff
@@ -165,6 +167,40 @@ def rapiddns(self, q):
 		j = self.page_parse(req.text).get_dns(q) or []
 		set_data(j)
 
+def dnsdumpster(self, q):
+	self.verbose('[DNSDUMPSTER] Searching in dnsdumpster...')
+	init_res = requests.get('https://dnsdumpster.com/')  # initial response from dnsdumpster
+	set_cookie = init_res.headers.get('Set-Cookie')  # getting crsftoken from cookie field
+	cookie = set_cookie[:set_cookie.index(';') + 1]
+	headers = {
```

Comment:
I think you don't need to add all of the fields to the headers.
Remove the unnecessary fields.